components/org.wso2.carbon.identity.api.server.cors/org.wso2.carbon.identity.api.server.cors.v1/src/main/resources/configs.yaml

```diff
@@ -0,0 +1,232 @@
+      operationId: patchCORSOrigins
+      description: A JSONPatch as defined by RFC 6902.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CORSOriginPatchRequest'
+        required: true
+      responses:
+        '200':
+          description: Successful Response
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /cors/origins/{cors-origin-id}/apps:
+    get:
+      tags:
+        - App associations
+      summary: Retrieve the Applications associated with a CORS origin by ID.
+      operationId: getAssociatedAppsByCORSOrigin
+      description: Retrieve the Applications associated with a CORS origin by ID.
+      parameters:
+        - name: cors-origin-id
+          in: path
+          description: ID of a CORS origin
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CORSOriginAppsGetResponse'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+servers:
+  - url: 'https://localhost:9443/t/{tenant-domain}/api/server/v1'
+    variables:
+      tenant-domain:
+        default: carbon.super
+components:
+  securitySchemes:
+    BasicAuth:
+      type: http
+      scheme: basic
+    OAuth2:
+      type: oauth2
+      flows:
+        authorizationCode:
+          authorizationUrl: 'https://localhost:9443/oauth2/authorize'
+          tokenUrl: 'https://localhost:9443/oauth2/token'
+          scopes: {}
+  parameters:
+    typeQueryParam:
+      in: query
+      name: type
+      required: false
+      description: |
+        Type of authenticators. Can be either 'LOCAL' or 'REQUEST_PATH'
```

Comment:
shall we define an enum for this?

Reply:
This part was not introduced through this PR.